src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -4422,6 +4422,44 @@ void MacroAssembler::sign_extend(Register dst, Register src, int bits) {
   }
 }
 
+void MacroAssembler::compress_bits(Register dst, Register src, Register mask, Register tmp, bool is_long) {
```

Comment:
Since `compress_bits` uses RVV instruction, I think we should add `_v` suffix.
```suggestion
void MacroAssembler::compress_bits_v(Register dst, Register src, Register mask, Register tmp, bool is_long) {
```